pubspec.yaml

```diff
@@ -11,23 +11,18 @@ authors:
 homepage: https://github.com/Workiva/w_flux
 
 environment:
-  sdk: '>=1.24.3 <3.0.0'
+  sdk: '>=2.4.0 <3.0.0'
 
 dependencies:
-  meta: ^1.0.4
-  react: ">=4.7.0 <6.0.0"
-  w_common: ^1.9.0
+  meta: ^1.1.7
+  react: ^4.9.2
+  w_common: ^1.20.1
 
 dev_dependencies:
-  build_runner: ">=0.6.0 <2.0.0"
-  build_test: ">=0.9.0 <2.0.0"
-  build_web_compilers: ">=0.2.0 <3.0.0"
-  coverage: ">=0.10.0 <0.13.0"
-  dart_dev: ^2.0.0
-  dart_style: ^1.0.9
-  over_react: ">=2.4.0 <4.0.0"
-  test: ">=0.12.30 <2.0.0"
-
-transformers:
-  - test/pub_serve:
-      $include: test/**_test{.*,}.dart
+  build_runner: ^1.7.1
+  build_test: ^0.10.9
+  build_web_compilers: ^2.5.1
+  dart_dev: ^3.0.0-alpha
```

Comment:
@evanweible-wf  what is the best way to go about `dart_dev` version?

Reply:
I'm hoping that we can get 3.0.0 stable released early next week, so let's just hold until then